custom_completions/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I'd remove this honestly! 

I'd prefer doing it the pnpm way.

```bash
pnpm completion bash > ~/completion-for-pnpm.bash
echo 'source ~/completion-for-pnpm.bash' >> ~/.bashrc
```

So we'd have:
```bash
npx @bombsh/tab pnpm zsh > ~/.tab.pnpm.zsh
echo 'source ~/.tab.pnpm.zsh' >> ~/.zshrc
```

+ Other shells (e.g. fish) and package managers (e.g. npm).

I'd love to hear your thought on better ways to do it.